fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/socialmedia/SocialMediaViewModel.java

```diff
@@ -226,13 +239,7 @@ public LaoView getLaoView(String laoId) throws UnknownLaoException {
     return laoRepository.getLaoView(laoId);
   }
 
-  @Nullable
-  public Lao getCurrentLao() throws UnknownLaoException {
-    return getLao(getLaoId().getValue());
-  }
-
-  @Nullable
-  private Lao getLao(String laoId) throws UnknownLaoException {
-    return laoRepository.getLaoView(laoId).createLaoCopy();
+  public LaoView getCurrentLaoView() throws UnknownLaoException {
+    return getLaoView(getLaoId().getValue());
```

Comment:
I think it would be good to do that same thing as in LaoDetailViewModel: Store the laoId and use it to retrieve the LaoView.

What do you think ?

Reply:
True. The mutable liveData is useless for how it is used